release-notes/Cloud/DCP_change_log_2025.md

```diff
@@ -0,0 +1,16 @@
+---
+uid: DCP_change_log_2025
+---
+
+# dataminer.services change log - 2025
+
+The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available in 2025.
+
+> [!NOTE]
+> Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
+
+### 9 January 2025 - Enhancement - Remote Access performance and stability [ID 41897]
+
+Several enhancements were made to improve performance and stability for all remote access features which have immediate effect.
+
+In addition dataminer.services now supports for the CloudGateway DxM >= 2.17.0 to create multiple connections instead of one to increase throughput and improve stability even more. CloudGateway 2.17.0, will take benefit of this, and is expected to be released in the upcoming weeks.
```

Comment:
```suggestion
In addition dataminer.services now supports for the CloudGateway DxM >= 2.17.0 to create multiple connections instead of one to increase throughput and improve stability even more. CloudGateway 2.17.0 will take benefit of this and is expected to be released in the upcoming weeks.
```